### PR TITLE
Optimize upgrade script.

### PR DESCRIPTION
**Pull request by smoshiur1237**
Optimize the upgrade script with removing unnecessary task.

### CODE REVIEW

**Review by Rozzii — APPROVED** [review]

**Comment by Rozzii** [comment]
LGTM

File: vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml [hunk 1]
```diff
@@ -474,28 +474,17 @@
...
-            kubectl get bmh -n {{NAMESPACE}} | grep -i provisioned | grep -c "{{ CLUSTER_NAME }}-controlplane-"
-    environment:
-      KUBECONFIG: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
-    retries: 200
-    delay: 20
-    register: upgraded_cp_nodes_count
-    until: upgraded_cp_nodes_count.stdout|int == 0
-    failed_when: upgraded_cp_nodes_count.stdout|int != 0
-
-  - name: Get control plane machines
```
Comment (kashifest) [inline vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml:488]: We don't need this check anymore?

Comment (smoshiur1237) [reply vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml:488]: line 477: We don't need that task as we are using scale-in feature, there is no old control plane in the scenario. 
line:488 I changed the name of that task only.

Comment (kashifest) [reply vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml:488]: Ah right

**Review by kashifest — APPROVED** [review]

**Pull request merged**